internal/commands/secrets/create_inputs.go

```diff
@@ -0,0 +1,29 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type createInputs struct {
+	app.ProjectInputs
+	Name  string
+	Value string
+}
+
+func (i *createInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if i.Name == "" {
+		if err := ui.AskOne(&i.Name, &survey.Input{Message: "Secret Name"}); err != nil {
+			return err
+		}
+	}
+	if i.Value == "" {
+		if err := ui.AskOne(&i.Value, &survey.Input{Message: "Secret Value"}); err != nil {
+			return err
+		}
+	}
+
+	return nil
```

Comment:
Out of curiosity, should we re-prompt if a user passes in an empty string? I don't believe we do elsewhere, and I actually like the idea of us not maintaining logic at this level of code. I just wasn't sure if it would be weird to perform the following steps:
1. Run `secrets create`
2. Press enter when prompted for `Secret Name: `(i.e. passing the empty string)
3. Press enter when prompted for `Secret Value`
4. Select an app if one is not provided
5. `secrets create` fails because the name and value are empty

Reply:
no we should fail in that scenario... we want a frictionless experience, but not some endless loop chasing certain inputs from the user.

if they are prompted for name and value of a secret and hit enter to leave them blank, we should respond with an error indicating "failed to create secret"